MetasysServicesExampleApp/FeaturesDemo/JsonOutputDemo.cs

```diff
@@ -1144,6 +1144,132 @@ private static void PrintMenu()
             Console.WriteLine("28) AddAuditAnnotationMultiple");
             Console.WriteLine("29) DiscardAuditMultiple");
             Console.WriteLine("99) Exit");
+        }
+
+        private static IEnumerable<OriginApplicationsEnum> GetOrgAppEnumItem(OriginApplicationsEnum enumerableItem)
+        {
+            IList<OriginApplicationsEnum> originApplications = new List<OriginApplicationsEnum>();
+
+            if (enumerableItem.HasFlag(OriginApplicationsEnum.AlarmEvent))

```

Comment:
```suggestion
           This could be improved iterating the Enum.GetValues(Type) instead of checking with a statement for every possible value, in this way a change on the enumset does not impact the logic here. 

```